src/parser/expression.js

```diff
@@ -301,6 +301,24 @@ export default class ExpressionParser extends LValParser {
         node.object = base;
         node.callee = this.parseNoCallExpr();
         return this.parseSubscripts(this.finishNode(node, "BindExpression"), startPos, startLoc, noCalls);
+
+      } else if (this.eat(tt.questionDot)) {
```

Comment:
I added the plugin check, updated the corresponding documentation and also added an explicit error message since it's a new syntax.